package.json

```diff
@@ -11,7 +11,7 @@
     "serve": "serve ./dist",
     "lint": "eslint .",
     "typecheck": "tsc --noEmit",
-    "test": "npm run typecheck && npm run lint"
+    "test": "mocha --timeout 10000 ./**/*.test.ts"
```

Comment:
We prefer to give each test its own timeout value.
some tests would need a smaller timeout, and some would possibly need a larger one.

heres a thread on stack overflow where they discuss how to do this
https://stackoverflow.com/questions/23492043/change-default-timeout-for-mocha